src/Sarif/Sarif.csproj

```diff
@@ -19,7 +19,8 @@
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), build.props))\build.props" />
 
   <PropertyGroup>
-    <TargetFrameworks>netstandard2.0;net461;net45</TargetFrameworks>
```

Comment:
>net461 [](start = 37, length = 6)

Removing `net45` is fine. I know @michaelcfanning has talked before about removing .NET Framework support altogether. Is that what's happening here? That he said "Go ahead and remove it"?

Reply:
Oh no, sorry, I see what's going on. Never mind.

---
In reply to: [507934421](https://github.com/microsoft/sarif-sdk/pull/2122#discussion_r507934421) [](ancestors = 507934421)